src/api/v1/db/oracledb/pizzas-dao.js

```diff
@@ -0,0 +1,93 @@
+import _ from 'lodash';
+
+import { getConnection } from 'api/v1/db/oracledb/connection';
+import { getDoughById } from 'api/v1/db/oracledb/doughs-dao';
+import { serializePizza } from 'api/v1/serializers/pizzas-serializer';
+
+const pizzaColumns = {
+  id: 'ID',
+  doughId: 'DOUGH_ID',
+  name: 'NAME',
+  bakeTime: 'BAKE_TIME',
+  ovenTemp: 'OVEN_TEMP',
+  specialInstructions: 'SPECIAL_INSTRUCTIONS',
+};
+
+const pizzaColumnAliases = _.map(pizzaColumns,
+  (columnName, attributeName) => `${columnName} AS "${attributeName}"`)
+  .join(', ');
+
+/**
+ * Fetch all the ingredients associated with the pizza with id
+ *  `pizzaId`
+ *
+ * @param {string} pizzaId
+ * @returns {Array} an array of unserialized ingredient objects
+ */
+const getPizzaIngredients = async (pizzaId) => {
+  const query = 'SELECT INGREDIENTS.ID AS "id", INGREDIENTS.TYPE AS "ingredientType", INGREDIENTS.NAME as "name", INGREDIENTS.NOTES AS "notes" FROM PIZZA_INGREDIENTS INNER JOIN INGREDIENTS ON PIZZA_INGREDIENTS.INGREDIENT_ID = INGREDIENTS.ID WHERE PIZZA_INGREDIENTS.PIZZA_ID = :id';
```

Comment:
Let's split this bad boy up. I think you can use backticks ` for multiline strings.